### DISCUSSION TITLE
Option to disable the size change animation of the combo counter in Lazer

### DISCUSSION THREAD

**Discussion by ItsCollector** [ppy/osu#28814] [Ideas]
This is just my opinion, bringing this forward as a mania player.

I find the combo animation very distracting and prefer non-animated UI elements where possible. Reason for it being distracting is because, the combo grows very quickly on mania due to the large amounts of notes in a short space of time. 

I couldn't speak for players who play other gamemodes since I don't play those so they may have a different opinion on this. 

Other VSRGs like LR2 have non-animated UI elements during a map and it creates a nice environment where players can focus heavily on reading the chart. 

Being able to at least toggle the animation on/off would keep the players who like the animation happy, while enhancing the gameplay experience for players like myself who prefer as little distractions as possible.

**Comment by SupDos** [comment]
I agree that having that as an option for the combo counter would be nice - for now though you can use the `DefaultComboCounter` in the skin editor to get what you're looking for:

https://github.com/ppy/osu/assets/6813986/487cb415-3531-4cc6-866b-5e7852b17390

**Reply by ItsCollector** [reply]
Thanks for this response. I didn't see the default combo counter in the editor and I will be definitely using this.
